lib/msal-node/test/network/HttpClient.spec.ts

```diff
@@ -199,44 +199,48 @@ describe("HttpClient", () => {
         });
     });
 
-    describe("Bad Status Code Error", () => {
+
+    describe("Bad Status Code Error", <T>() => {
```

Comment:
```suggestion
    describe("Bad Status Code Error", () => {
```

Don't think this change is needed?

Reply:
Needed for L206

Reply:
That isn't the right way to pass a type for that. See below.